Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageComposerView.swift

```diff
@@ -190,4 +198,8 @@ open class ChatChannelMessageComposerView<ExtraData: ExtraDataTypes>: UIInputVie
     }
     
     open func updateContent() {}
+    
+    open func setCheckmarkView(hidden: Bool) {
+        container.bottomStackView.isHidden = hidden
```

Comment:
👍🏼 nice

Reply:
Good catch! Tbh that's strange that we haven't faced any issues with that so far  🤔🤔

Reply:
I guess yeah, just flashback :) So its okay then :)